app/src/main/java/com/yeonae/chamelezone/view/home/presenter/HomeContract.kt

```diff
@@ -1,19 +1,24 @@
 package com.yeonae.chamelezone.view.home.presenter
 
+import com.yeonae.chamelezone.data.model.LikeStatusItem
 import com.yeonae.chamelezone.network.model.PlaceResponse
 import com.yeonae.chamelezone.network.room.entity.UserEntity
 
 interface HomeContract {
 
     interface View {
         var presenter: Presenter
-        fun showHomeList(response: List<PlaceResponse>)
+        fun showHomeList(place: List<PlaceResponse>)
         fun getMember(user: UserEntity)
         fun getMemberCheck(response: Boolean)
+        fun showLikeMessage(response: LikeStatusItem)
```

Comment:
```suggestion
        fun showLikeMessage(likeStatusItem: LikeStatusItem)
```
response를 가공한 데이터가 ~Item class 인데,
response 라는 네이밍은 의미에 맞지 않다고 생각이 듭니다.
전체적으로 해당하는 부분을 점검해보는것이 좋겠어요.

Reply:
수정했습니다
0d15d0d